### ISSUE TITLE
namebench should have a way to share results

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#105]
```
It would be handy for DNS service providers to have a way to see how poorly
their DNS service was doing around the world. For instance, global
providers such as Google, UltraDNS, OpenDNS might like to know that their
service is completely useless in Africa compared to local DNS servers.

There should be a way to opt-in and submit a set of anonymized results to a
central public resource. Something that supplies basic information such as:

* List of DNS servers
* min, max, average for DNS servers
* durations for 1-3 reference hostnames (hardcoded). Innocuous hosts from
the Alexa Top 10 domains. 

We could submit this to a central site via HTTP POST, in YAML or some other
format - so long as the format is user readable, so that the user can OK
it's transmission. It should also filter out any private DNS servers.

[instance]
class_c=208.11.244
date=2010-01-24

[nameservers]
# IP = min, average, max, 
24.244.65.8=0.19,53.0,
24.244.65.9=
24.247.15.53=
24.247.24.53=
24.25.195.1=

[reference]
live.com
www.nytimes.com
www.youtube.com

```

Original issue reported on code.google.com by `tstromb...@google.com` on 2 Mar 2010 at 2:44

**Comment by GoogleCodeExporter** [comment]
```
It may actually be useful to include query durations for all queries in this, 
so that
we can see duration graphs. In order to protect user privacy, we cannot however
include the hostnames associated to these durations.

```

Original comment by `tstromb...@google.com` on 2 Mar 2010 at 2:45

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3b1.

```

Original comment by `helixblue` on 20 May 2010 at 3:25
- Changed state: **Fixed**
